cmd/example-noisy-tests/main.go

```
@@ -0,0 +1,87 @@
+// realistic non-JSON garbage to stdout when invoked as a run-test subprocess.
+// This exercises the extractJSON parsing pipeline end-to-end, ensuring that
+// klog lines, Ginkgo reporter text, and other non-JSON output that commonly
+// appears on stdout in extension binaries does not break result deserialization.
+func main() {
+	// When this binary is spawned as a subprocess by SpawnProcessToRunTest
+	// (i.e. `run-test` is the first argument), emit realistic non-JSON content
+	// to stdout before the cobra command runs. This simulates the kind of
+	// stdout contamination that occurs in practice from klog, Ginkgo reporters,
+	// and other libraries that write to stdout.
+	if len(os.Args) > 1 && os.Args[1] == "run-test" {
+		fmt.Fprintln(os.Stdout, `I0625 19:25:00.000000   12345 client.go:123] Connecting to apiserver...`)
```

golangci-lint (2.12.2)

[error] Error return value of `fmt.Fprintln` is not checked

(errcheck)